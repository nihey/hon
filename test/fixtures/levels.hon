bash
    bash
        bash
            bash
                bash
                    bash
                        bash
                            bash
                                bash
                                    bash
                                        bash
                                            bash
                                                echo "The blood makes us human, makes us more than human, makes us human no more."
                echo "Praise the sun!"
