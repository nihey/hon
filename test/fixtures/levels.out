The blood makes us human, makes us more than human, makes us human no more.
Praise the sun!
